app/views/publishers/create_auth_token.html.slim

```diff
@@ -4,8 +4,8 @@
 .single-panel--wrapper
   .single-panel--content
     .row
-      h3.single-panel--headline = t("publishers.create_auth_token")
+      h3.single-panel--headline= t ".heading"
```

Comment:
Why no space after the class?

Reply:
It works either way; that's just how I've always written my Slim tags with computed contents I guess?

Reply:
I would like to pick a consistent style. I don't care much, but I've been using spaces after the class.

I just don't want to think about it reviewing diffs. @ayumi any input?

Reply:
The existing templates were also using no-space as the default before we entered the project.

Reply:
I'm happy to go this direction! Wish we could lint it 😭